packages/shared/styles/components/molecules/SfBar.scss

```diff
@@ -3,21 +3,21 @@
 .sf-bar {
   box-sizing: border-box;
   display: flex;
-  justify-content: space-between;
   align-items: center;
   height: var(--bar-height, 3.125rem);
-  padding: var(--bar-padding, 0 var(--spacer-big));
-  background-color: var(--bar-background, var(--c-light));
-  font: var(
+  padding: var(--bar-padding, 0 var(--spacer-regular));
+  background-color: var(--bar-background, #f2f2f2);
+  @include font(
     --bar-font,
-    font(
-      --bar,
-      500,
-      var(--font-size-regular),
-      1.6,
-      var(--body-font-family-secondary)
-    )
+    var(--font-weight-regular),
+    var(--font-size-regular),
```

Comment:
In designs font-weight is 500 -> currently its awaiting Marta's PR, but you can set is as var(--font-weight-bold)